### PR TITLE
Structure the to-do list

### PR DESCRIPTION
**Pull request by kennedyowusu**
### I implemented the following

- [ ] Used ES6 syntax in creating some logic in the js files
- [ ] Used ES6 Modules to write modular code
- [ ] Used webpack to bundle js
- [ ] Placed index.html, index.js, and style.css files in the _/src_ directory

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      
+    </div>
+    
+  </li>
+  <hr class="divider" />
+  `;
+  document.querySelector('.todo-list').innerHTML += dataPlaceHolder;
+});
+
+window.onload = renderUIData;
```
Comment (mckent05) [inline src/index.js]: **OPTIONAL** - Good job on the work done so far 👏 . I suggest it is cleaner to attach or call your `renderUIData` function to an event listener like `DOMContentLoaded`. Please fix this 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @kennedyowusu,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there


## Highlights
- All linter checks are passing ✔️ 
- Your PR is professional 👏  


## Required Changes ♻️
- I cannot run your project well locally, please check that you have properly configured your webpack file
hrhrhr 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (kennedyowusu) [reply src/index.js]: Thanks. I will do that

**Comment by kennedyowusu** [comment]
Resolved the issue raised by @mckent05 Image of the solution is below: 


todo-with-webpack


I configured the project to run on port:3000. I like to use that port for frontend-related works. 

port3000

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @kennedyowusu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**